README.md

```diff
@@ -51,12 +51,18 @@ By default, the variant is `frappe`, the accent is `blue`, the size is `standard
 
 Example:
 ```nix
-pkgs.catppuccin-gtk.override {
-  accents = [ "pink" ]; # You can specify multiple accents here to output multiple themes
-  size = "compact";
-  tweaks = [ "rimless" "black" ]; # You can also specify multiple tweaks here
-  variant = "macchiato";
-}
+# as per wiki examples
```

Comment:
Can you add a link to the wiki page this is from?

Reply:
i used an example from nixos.wiki on vencord with discord overriding. https://nixos.wiki/wiki/Discord